app/javascript/components/track/iteration-summary/AnalysisStatusSummary.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { RepresentationStatus, AnalysisStatus } from '../IterationSummary'
+
+function Content({
+  analysisStatus,
+  representationStatus,
+}: {
+  analysisStatus: AnalysisStatus
+  representationStatus: RepresentationStatus
+}) {
+  if (
+    analysisStatus === AnalysisStatus.QUEUED ||
+    representationStatus === RepresentationStatus.QUEUED
+  ) {
+    return (
+      <>
+        <svg role="presentation" className="icon">
+          <use xlinkHref="#loading" />
+        </svg>
+        <div className="status">Analysing</div>
+      </>
+    )
+  }
+
+  return (
+    <>
+      <div className="dot"></div>
+      <div className="status">Analysed</div>
+    </>
+  )
+}
+
+export function AnalysisStatusSummary({
+  analysisStatus,
+  representationStatus,
+}: {
+  analysisStatus: AnalysisStatus
+  representationStatus: RepresentationStatus
+}) {
+  return (
+    <div className="analyzer">
```

Comment:
I'm not entirely sure about our conventions, but didn't we prefix components with `c-`?

Reply:
⏰  NEW CONVENTION ALERT ⏰ 

All class names **within** a component (of which this is an example, I think?) should be prefixed with `--` So this would be `--analyzer`.

If this was considered it's own component it would be `c-` (which requires a standalone stylesheet file etc) but as it's correctly only used as part of the iteration summary, I think we should consider it to be a subcomponent and use `--...`.

Reply:
This was just copied from existing CSS that we have.